### PR TITLE
feat: migrate helm release

### PR DESCRIPTION
**Pull request by arunjaindev**
# Description
Add tab to migrate helm release in CD

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] Does this PR require documentation updates?
- [ ] I've updated documentation as required by this PR.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Pages/CDPipeline/types.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import { DeploymentAppTypes } from '@Common/Types'
+import { DeploymentStrategy } from '@Shared/Components'
+
+interface ConfigSecretType {
+    label: string
+    value: string
+    type: string
+}
+
+export interface Environment {
```
Comment (AbhishekA1509) [inline src/Pages/CDPipeline/types.tsx:11]: Check if can use EnvListMinDTO

File: src/Pages/CDPipeline/types.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    active: boolean
+    clusterName: string
+    clusterId: string
+    isClusterCdActive: boolean
+    isVirtualEnvironment?: boolean
+    allowedDeploymentTypes?: DeploymentAppTypes[]
+    isDigestEnforcedForEnv?: boolean
+}
+
+export interface PipelineBuildStageType {
```
Comment (AbhishekA1509) [inline src/Pages/CDPipeline/types.tsx:25]: can extend BuildStageType or we can update BuildStageType if same

**Review by AbhishekA1509 — CHANGES_REQUESTED** [review]

File: src/Pages/CDPipeline/types.tsx (hunk 1 above)
Comment (arunjaindev) [reply src/Pages/CDPipeline/types.tsx:11]: some keys are different

Comment (AbhishekA1509) [reply src/Pages/CDPipeline/types.tsx:11]: Can atleast Pick same types

**Review by AbhishekA1509 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**